src/Analysis/GWASV2/Steps/SelectOutcome/SelectOutcome.jsx

```diff
@@ -11,6 +11,9 @@ const SelectOutcome = ({ dispatch, studyPopulationCohort, outcome }) => {
     if (selectionMode === 'continuous') {
       return (
         <ContinuousCovariates
+          selectedStudyPopulationCohort={studyPopulationCohort}
+          selectedCovariates={[]} // TODO - add to props above as well and pass in here...
```

Comment:
actually I'm not so sure about whether we should really DO this TODO. I'll log a follow-up ticket for discussion